src/UriTemplate.php

```diff
@@ -50,7 +50,7 @@ final class UriTemplate
     ];
 
     /**
-     * @var array<int, string> Percent encoded delimiters
+     * @var string[] Percent encoded delimiters
```

Comment:
why is this better?

Reply:
It's not better, it's just you'd used that notation everywhere else apart from there, for integer indexed arrays.